CONTRIBUTING.md

```diff
@@ -47,24 +50,32 @@ language or language that is a matter of preferred usage are not considered
 nits.
 
 An example of preferential language:
```

Comment:
Not quite related to the direct changes of this PR, but i think while we're at it, it may be easier to read if we have subheaders for preferential language vs nits
```suggestion
##### An example of preferential language:
```